conserver/inside-the-conserver.md

```diff
@@ -69,7 +69,7 @@ The [main loop of the conserver](https://github.com/vcon-dev/vcon-server/blob/86
 * When an item (vCon ID) is found, it creates a VconChainRequest and processes it.
 * Handles exceptions by moving problematic vCons to a Dead Letter Queue.
 
-Step by step:&#x20;
```

Comment:
Are the encoded spaces a gitbook issue, or just an anomaly for local editing, submitted to github?